scripts/algcomptool/tool/AlgCompTool.py

```diff
@@ -0,0 +1,317 @@
+#Author: Mihai Avram - mihai.v.avram@gmail.com
+
+import sys, getopt
+import pandas as pd
+import numpy as np
+import string
+
+    
+def main(argv):
+    if len(argv) < 2:
+        print("Executing...\n")
+        computeAndPrint(False)
+    elif(argv[1] in ['--help','-h']):
+        print("Enter python AlgCompTool.py to print to terminal, or python AlgCompTool.py -f to print to files")
+    elif(argv[1] in ['--file','-f']):
+        print("Executing...\n")
+        computeAndPrint(True)
+
+
+def computeAndPrint(writeToFile):
+    #Importing algorithms to be compared
+    alg1 = pd.read_csv("../input/presidentcouplesNODES.csv")
```

Comment:
Input file should not be hardcoded into source code. Should be passed as command line arguments instead.